sfs/util.py

```diff
@@ -555,6 +556,28 @@ def spherical_hn2(n, z):
     return spherical_jn(n, z) - 1j * spherical_yn(n, z)
 
 
+def jinc(x):
+    r"""Circular counterpart of Sinc function a.k.a. Jinc function
+
+    .. math::
+
+        \mathrm{Jinc}(x) = \frac{J_1(x)}{x}
+
+    where :math:`J_1(\cdot)` is the Bessel function of first order,
+    and :math:`x` its real-value argument.
+
+    Parameters
+    ----------
+    x : array_like
+        Argument of the Jinc function.
+
+    """
+
+    x = np.asanyarray(x)
+    y = where(x == 0, 1.0e-20, x)
```

Comment:
I can't tell if this is the right thing to do.

Is there a reference implementation or a literature reference you could mention?